acceptance/config/beaker/options.rb

```diff
@@ -11,5 +11,5 @@
  "puppetserver-confdir"=>"/etc/puppetlabs/puppetserver/conf.d",
  "puppetserver-config"=>
   "/etc/puppetlabs/puppetserver/conf.d/puppetserver.conf",
- :puppet_version=>"5.3.2.25.g1b6a5aa",
- :puppet_build_version=>"5f10b8f895ab925b92740104c006f6acb4dc3a1a"}
+ :puppet_version=>"5.3.2.66.g0ff9cf7",
+ :puppet_build_version=>"ac142bea60e5ce29efc52b12a9969c789b2ef15f"}
```

Comment:
Where did you get these versions? According to Kerminator, there is a newer passing build at puppet-agent b6f32bc9a276153a8fa6177b07789b138365d3c9 and puppet 5.3.2.76.g936a959.